Makefile

```diff
@@ -5,3 +5,11 @@ PHONY: \
 
 build_test_image:
 	$(DOCKER_COMPOSE) -f docker-compose.yml  run --rm postgres -d
+
+test:
+	go generate ./...
+	docker-compose -f testdata/docker-compose.yml up --abort-on-container-exit --remove-orphans
```

Comment:
Is there a reason to use `$(DOCKER_COMPOSE)` on `make build_test_image`, but `docker-compose` on `test` and `test_fileoutput`?

Reply:
Fixing.